core/modules/filter/filter.module

```diff
@@ -1247,7 +1247,7 @@ function filter_process_format($element) {
   $element['format'] = array(
     '#type' => 'fieldset',
     '#attributes' => array('class' => array('filter-wrapper')),
-    '#title' => t('Formatting options'),
+    '#title' => t('Filtering options'),
```

Comment:
Can we please leave this change out for 1.12.2 and stick to only the style changes? We can flesh this out for 1.12.3

Reply:
yep :)